src/utils/getQueryParameters.ts

```diff
@@ -0,0 +1,11 @@
+const getQueryParameters = (search: string | null) =>
```

Comment:
Not supported by your router lib?

Reply:
I agree should be ```new URLSearchParams(useLocation().search)``` .get("whatever_query_param")